tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -514,6 +551,22 @@ def sync_volumes(volumes, index, vol_options):
                     'volume%s.brick%s.is_arbiter' % (index, b_index)
                 ),
             )
+            if not brick.exists():
+                job_id = monitoring_utils.update_dashboard(
+                    brick.brick_name,
+                    RESOURCE_TYPE_BRICK,
+                    NS.tendrl_context.integration_id,
+                    "add"
+                )
+                Event(
+                    Message(
+                        priority="info",
```

Comment:
keep the priority as "debug" for all these logs, "info" is for user facing messages only and they put load on etcd. You know this issue already.

Reply:
i will change this